### DISCUSSION TITLE
Welcome to miney!

### DISCUSSION THREAD

**Discussion by Netzvamp** [miney-py/miney#3] [General]
## 👋 Welcome!

I'm the developer of miney and i'm looking for support. If someone is interested, please contact me.

**Comment by ChynamaE** [comment]
Hello I am ChynamaE of the Fone Network. I am here to support you with anything you may need assistance with. I am not a developer but I do have a large fan base on my NFT platform. Come by make a profile and create your own NFT to publish for Solana SPL tokens called, "Fone" it is still a far shot from perfect and has a lot of fake accounts from malicious developers but with help from others who are app developers maybe we can get this ball rolling and start a successful approach to application development together! EMAIL ME AT: "nattrass92@gmail.com

https://play.google.com/store/apps/details?id=com.cloud.earning  and use my refferal please and thank you the code is: "uuky8adf
